src/Adv/AdvModel.php

```diff
@@ -452,7 +452,9 @@ public function GetAdsDefaultCoverImageByAdId($id)
     public function viewed_Ad($id)
     {
         $ad = $this->find($id);
-        $ad->update(['count_show_ad' => intval($ad->count_show_ad) + 1]);
+        if ($ad) {
```

Comment:
@emrullahardc bu zaten model de. Bunu this ile setAttribute yapar mısın? Kullanılan yerde adv sorgulanmalıdır.